src/main.js

```diff
@@ -54,19 +52,17 @@ Apify.main(async () => {
         }
     });
 
-
-    // Puppeteer options
-    const launchPuppeteerOptions = proxyConfig || {};
-    if (liveView) {
-        launchPuppeteerOptions.liveView = true;
-    }
-    launchPuppeteerOptions.stealth = true;
-    launchPuppeteerOptions.useChrome = true;
+    const proxyConfiguration = await Apify.createProxyConfiguration(proxyConfig);
 
     // Create the crawler
     const crawlerOptions = {
         requestList,
         requestQueue,
+        proxyConfiguration,
+        launchContext: {
+            useChrome: true,
+            stealth: true,
```

Comment:
both settings can be removed and become:

```js
launchContext: {
   useIncognitoPages: true,
},
browserPoolOptions: {
   useFingerprints: true,
}
```
the `stealth` was a way to hide some chromium detection, that isn't needed anymore